src/entries/background/index.ts

```diff
@@ -40,3 +62,10 @@ popupMessenger.reply('rainbow_updateWagmiClient', async () => {
   const { rainbowChains } = getRainbowChains();
   updateWagmiConfig(rainbowChains);
 });
+
+browserAPI.commands.onCommand.addListener((command: string) => {
```

Comment:
Could be crazy, but thought `chrome` mapped one to one to `browser`, so thought `chrome.commands` should work in Firefox. If so, I think we could reduce the need for the polyfill and just do a check in the `openPopup` wrapper function for whichever of `browserAction` or `action` is defined and call the appropriate one

Reply:
okay i think youre right actually.